hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -8,6 +8,8 @@ parameters:
   required: true
 - name: IMAGE_TAG
   required: true
+- name: REPO_DIGEST
```

Comment:
Are the `REGISTRY_IMG`, `CHANNEL` and `IMAGE_TAG` parameters on this template still needed?

Reply:
It seems only `IMAGE_TAG` is still used

Reply:
Yes. They're used under the covers to construct and discover the REPO_DIGEST. [Doc](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/continuous-delivery-in-app-interface.md#automatically-generated-parameters)